--- origsrc/gitg-0.0.7/libgitg/gitg-runner.c
+++ src/gitg-0.0.7/libgitg/gitg-runner.c
@@ -34,6 +34,13 @@
 #include <gio/gunixoutputstream.h>
 #include <gio/gunixinputstream.h>
 
+#ifdef stdin
+#undef stdin
+#endif
+#ifdef stdout
+#undef stdout
+#endif
+
 #define GITG_RUNNER_GET_PRIVATE(object)(G_TYPE_INSTANCE_GET_PRIVATE((object), GITG_TYPE_RUNNER, GitgRunnerPrivate))
 
 struct _GitgRunnerPrivate
