--- origsrc/gitg-0.0.4/gitg/gitg-runner.c	2009-07-05 13:29:41.000000000 -0500
+++ src/gitg-0.0.4/gitg/gitg-runner.c	2009-07-10 10:12:57.425926400 -0500
@@ -569,8 +569,8 @@ gitg_runner_run_with_arguments(GitgRunne
 {
 	g_return_val_if_fail(GITG_IS_RUNNER(runner), FALSE);
 
-	gint stdout;
-	gint stdin;
+	gint std_out;
+	gint std_in;
 
 	gitg_runner_cancel(runner);
 
@@ -583,8 +583,8 @@ gitg_runner_run_with_arguments(GitgRunne
 	                                        NULL, 
 	                                        NULL,
 	                                         &(runner->priv->pid), 
-	                                         input ? &stdin : NULL, 
-	                                         &stdout, 
+	                                         input ? &std_in : NULL, 
+	                                         &std_out, 
 	                                         NULL, 
 	                                         error);
 
@@ -598,9 +598,9 @@ gitg_runner_run_with_arguments(GitgRunne
 	GOutputStream *output_stream = NULL;
 
 	if (input)
-		output_stream = G_OUTPUT_STREAM(g_unix_output_stream_new(stdin, TRUE));
+		output_stream = G_OUTPUT_STREAM(g_unix_output_stream_new(std_in, TRUE));
 		
-	input_stream = G_INPUT_STREAM(g_unix_input_stream_new(stdout, TRUE));	
+	input_stream = G_INPUT_STREAM(g_unix_input_stream_new(std_out, TRUE));	
 	ret = gitg_runner_run_streams(runner, input_stream, output_stream, input, error);
 	
 	if (output_stream)
