This should be handled by a configure switch.

--- origsrc/gitg-0.0.7/configure.ac	2010-12-29 09:19:04.000000000 -0600
+++ src/gitg-0.0.7/configure.ac	2010-12-29 20:18:27.044701200 -0600
@@ -51,7 +51,7 @@ AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GET
 
 AC_CHECK_LIB([m], [sinf])
 
-PKG_CHECK_EXISTS([gtk+-3.0 >= 2.90],
+PKG_CHECK_EXISTS([gtk+-3.0 >= 3.0],
 		 [
 			GTK_REQUIRED=gtk+-3.0
 			GTK_REQUIRED_VERSION=2.90.0
