--- origsrc/gitg-0.2.7/gitg/gitg-commit-view.c	2013-04-05 06:14:51.000000000 -0500
+++ src/gitg-0.2.7/gitg/gitg-commit-view.c	2014-03-25 08:34:23.289827600 -0500
@@ -24,6 +24,7 @@
 #include <gtksourceview/gtksourcebuffer.h>
 #include <gtksourceview/gtksourcelanguagemanager.h>
 #include <gtksourceview/gtksourcemarkattributes.h>
+#include <gtksourceview/gtksourcestylescheme.h>
 #include <gtksourceview/gtksourcestyleschememanager.h>
 #include <gtksourceview/gtksourcegutter.h>
 #include <gtksourceview/gtksourcegutterrendererpixbuf.h>
